array-methods.js

```diff
@@ -58,20 +100,119 @@ var stateSums = null;
     round this number to the nearest dollar before moving on.
   )
  */
-var sumOfHighInterests = null;
+var sumOfHighInterests = 0;
+
+let stateSum = [];
+
+dataArray.filter(
```

Comment:
Merge this line and the line above, and look into using reduce instead of forEach. All your methods should be chained together. Please make sure the rest of the file follows this format.